servicex/transformer/kafka_messaging.py

```diff
@@ -62,6 +72,6 @@ def publish_message(self, topic_name, key, value_buffer):
                                value=msg_bytes)
             self.producer.flush()
         except Exception as ex:
-            print("Exception in publishing message", ex)
+            self.logger.error(f"Exception in publishing message {ex}")
```

Comment:
Can transformer code use f-strings if we need to support Python 2.7?

Reply:
No, it can't.  But that's limited to transformers that run under ATLAS R21.  I'm not sure it'll be applicable for transformers in general

Reply:
Right, but would it cause issues if this code is imported and run by those transformers?

Reply:
FWIW, `setup.py` in this repo claims to support Python 2.7.